src/sentry/spans/consumers/process_segments/convert.py

```diff
@@ -75,6 +75,13 @@ def convert_span_to_item(span: CompatibleSpan) -> TraceItem:
     except Exception:
         sentry_sdk.capture_exception()
 
+    if span_meta := span.get("_meta"):
+        for attr, meta in (span_meta.get("attributes") or {}).items():
```

Comment:
In a follow up, can also think about making it work for top level fields (e.g. you'd see clock drift correction for timestamps)